### ISSUE TITLE
Package manager identifies warnings based on dependencies

### ISSUE THREAD

**Issue by BurtHarris** [Debdut/browser-extension#3]
npm install generates "high" vulnerabilities, but npm often creates false positives. Results when I tried:

```
npm WARN deprecated babel@6.23.0: In 6. x, the babel package has been deprecated in favor of babel-cli. Check https://opencollective.com/babel to support the Babel maintainers
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated uuid@3.4.0: Please upgrade to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142

added 1203 packages, and audited 1204 packages in 23s

116 packages are looking for funding
  run `npm fund` for details

12 vulnerabilities (9 moderate, 3 high)
```

I also tried with yarn as it usually generates fewer false positives.   Yarn said:

```
warning " > autoprefixer@10.4.7" has unmet peer dependency "postcss@^8.1.0".
warning " > postcss-loader@6.2.1" has unmet peer dependency "postcss@^7.0.0 || ^8.0.1".
warning " > tailwindcss@3.1.4" has unmet peer dependency "postcss@^8.0.9".
warning "web-ext > addons-linter > addons-scanner-utils@6.3.0" has unmet peer dependency "@types/download@8.0.1".
warning "web-ext > addons-linter > addons-scanner-utils@6.3.0" has unmet peer dependency "body-parser@1.19.1".
warning "web-ext > addons-linter > addons-scanner-utils@6.3.0" has unmet peer dependency "download@8.0.0".
warning "web-ext > addons-linter > addons-scanner-utils@6.3.0" has unmet peer dependency "express@4.17.2".
warning "web-ext > addons-linter > addons-scanner-utils@6.3.0" has unmet peer dependency "safe-compare@1.1.4".
```

**Comment by BurtHarris** [comment]
npm audit fix did minor safe (patch level) updates to clean up some of the warnings
down to 9 moderate and 1 high vulnerability.

**Comment by BurtHarris** [comment]
```
npm audit fix -force
npm WARN using --force Recommended protections disabled.
npm WARN audit Updating web-ext to 7.1.0, which is a SemVer major change.
npm WARN deprecated core-js@3.22.8: core-js@<3.23.3 is no longer maintained and not recommended for usage due to the number of issues. Because of the V8 engine whims, feature detection in old core-js versions could cause a slowdown up to 100x even if nothing is polyfilled. Some versions have web compatibility issues. Please, upgrade your dependencies to the actual version of core-js.
```

**Comment by Debdut** [comment]
Can you send in a PR?

**Comment by ChadBailey** [comment]
This can be closed, was fixed in PR #5
